packages/blade/src/components/Table/TableBody.web.tsx

```diff
@@ -217,6 +226,135 @@ const TableCell = assignWithoutSideEffects(_TableCell, {
   componentId: ComponentIds.TableCell,
 });
 
+const StyledEditableCell = styled(StyledCell)<{
+  rowDensity: NonNullable<TableProps<unknown>['rowDensity']>;
+}>(({ theme, rowDensity }) => ({
+  '&&&': {
+    '&:focus-visible': { outline: '1px solid' },
+    '&:focus-within': {
+      ...(rowDensity !== 'comfortable' ? getFocusRingStyles({ theme, negativeOffset: true }) : {}),
```

Comment:
bit hard to read inverted cases

```suggestion
      ...(rowDensity == 'comfortable' ? {} : getFocusRingStyles({ theme, negativeOffset: true })),
```